src/btree.h

```diff
@@ -341,26 +346,27 @@ btree_pack_req_init(btree_pack_req  *req,
    req->hash       = hash;
    req->seed       = seed;
    if (hash != NULL && max_tuples > 0) {
-      req->fingerprint_arr =
-         TYPED_ARRAY_ZALLOC(hid, req->fingerprint_arr, max_tuples);
+
+      fingerprint_init(&req->fingerprint, hid, max_tuples); // Allocates memory
```

Comment:
Inline memory allocation on old L345 is, henceforth, replaced by init()'ing the fingerprint object ... And so on ...